fe1-web/src/features/evoting/screens/CreateElection.tsx

```diff
@@ -285,6 +285,24 @@ const CreateElection = () => {
         {STRINGS.election_create_add_question}
       </PoPTextButton>
 
+      {!isConnected && (
+        <Text style={[Typography.paragraph, Typography.error]}>
+          {STRINGS.event_creation_must_be_connected}
+        </Text>
+      )}
+      {electionName === '' && (
+        <Text style={[Typography.paragraph, Typography.error]}>
+          {STRINGS.event_creation_name_not_empty}
+        </Text>
+      )}
+      {questions.some(isQuestionInvalid) && (
+        <Text style={[Typography.paragraph, Typography.error]}>
+          {STRINGS.election_create_invalid_questions_1 +
+            MIN_BALLOT_OPTIONS +
+            STRINGS.election_create_invalid_questions_2}
+        </Text>
+      )}
```

Comment:
We might also want to catch the case where there are multiple questions with the same name as this will cause question id collisions..

> "Question ID: Hash : HashLen('Question', election_id, question)"

(https://github.com/dedis/popstellar/blob/master/docs/messageData.md#setting-up-an-election-electionsetup)

Reply:
Okay, I'm ont it!